tests/test_writer.py

```diff
@@ -44,5 +43,5 @@ def test_register_generator(self):
     def test_write(self):
         with TemporaryDirectory() as tmpdir:
             writer.register_format(self.FAKE_NAME, FakeGenerator(tmpdir))
-            writer.write(Schema.create(), [], "", "fake")
+            writer.write([], "", "fake")
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Too many positional arguments for method call](https://app.codacy.com/manual/tefra/xsdata/pullRequest?prid=5141157)